migrations/versions/fbd510d51492_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: fbd510d51492
```

Comment:
For what it's worth, I would commit your migration in the same commit as the change to the model took place, so that it's impossible to have your commits and database out of sync.